azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java

```diff
@@ -1526,6 +1526,19 @@ WithManagedCreate withNewDataDiskFromImage(int imageLun,
             WithCreate withCustomData(String base64EncodedCustomData);
         }
 
+        /**
+         * The stage of the virtual machine scale set definition allowing to specify the custom data.
```

Comment:
`allowing to specify the custom data` -> `allowing to specify the secrets`

Reply:
Thanks, addressed.